tests/stream_tests.rs

```diff
@@ -0,0 +1,39 @@
+pub mod common;
+
+pub use common::{bakery_chain::*, setup::*, TestContext};
+pub use sea_orm::entity::*;
+pub use sea_orm::{QueryFilter, ConnectionTrait};
+
+#[sea_orm_macros::test]
+#[cfg(any(
+    feature = "sqlx-mysql",
+    feature = "sqlx-sqlite",
+    feature = "sqlx-postgres"
+))]
+pub async fn stream() {
+    use futures::StreamExt;
+
+    let ctx = TestContext::new("stream").await;
+
+    let bakery = bakery::ActiveModel {
+        name: Set("SeaSide Bakery".to_owned()),
+        profit_margin: Set(10.4),
+        ..Default::default()
+    }
+    .save(&ctx.db)
+    .await
+    .expect("could not insert bakery");
+
+    let result = Bakery::find_by_id(bakery.id.clone().unwrap())
+       .stream(&ctx.db)
+        .await
+        .unwrap()
+        .next()
+        .await
+        .unwrap()
+        .unwrap();
```

Comment:
Can we replace some of the `unwrap()` with `?` :P

Reply:
It's only one unwrap more than the `find_by_id_with_result` test that I've used as base for this test :P
TTo use `?` I have to make a test that returns `Result`, is it ok?

Reply:
Thanks!!